pvlib/losses.py

```diff
@@ -85,3 +86,128 @@ def soiling_hsu(rainfall, cleaning_threshold, tilt, pm2_5, pm10,
     soiling_ratio = 1 - 0.3437 * erf(0.17 * accum_mass**0.8473)
 
     return soiling_ratio
+
+
+def soiling_kimber(rainfall_timeseries, threshold=6, soiling_rate=0.0015,
+                   grace_period=14, max_soiling=0.3, manual_wash_dates=None,
+                   initial_soiling=0):
+    """
+    Kimber soiling model [1]_ assumes soiling builds-up at a daily rate unless
+    the daily rainfall is greater than a threshold. The model also assumes that
+    if daily rainfall has exceeded the threshold within a grace period, then
+    the ground is too damp to cause soiling build-up. The model also assumes
+    there is a maximum soiling build-up. Scheduled manual washes and rain
+    events are assumed to reset soiling to zero.
+
+    Parameters
+    ----------
+    rainfall_timeseries : pandas.Series
+        a timeseries of rainfall in millimeters
+    threshold : float, default 6
```

Comment:
```suggestion
    cleaning_threshold: float, default 6
```

Reply:
Would `rain_threshold` be okay here? I'm a tiny bit concerned that users will somehow confuse this with manual washes, so I'm looking for some way to clarify that this is the threshold of rain required to clean the panels. Any suggestions?

Reply:
leaving it as `cleaning_threshold` for consistency with `soiling_hsu` that already defines this term, also updated definition to be consistent and define accumulation period as one day